src/components/Search/CompareWithBase.tsx

```diff
@@ -290,15 +280,14 @@ function CompareWithBase({
             onRepositoryChange={(repo: Repository['name']) =>
               setBaseRepository(repo)
             }
+            formIsDisplayed={formIsDisplayed}
           />
           <SearchComponent
             {...stringsNew}
             isBaseComp={false}
             hasEditButton={hasEditButton}
             isWarning={isWarning}
             displayedRevisions={newInProgressRevs}
-            onSave={handleSaveNew}
-            onCancel={handleCancelNew}
             onEdit={handleEditNew}
```

Comment:
there's no need for onEdit on this SearchComponent now, you can pass an empty function `()=>{}` and remove `handleEditNew`.

Reply:
Yes, I planned to handle this in a new PR.